functions/src/api/middlewares/authenticateJWTMiddleware.spec.ts

```diff
@@ -0,0 +1,127 @@
+// tslint:disable-next-line
+import '../../testUtils/mockFirebaseAdminAndFunctions'
+import { firestoreGet } from '../../testUtils/firestoreStub'
+
+import { authenticateJWTMiddleware } from './authenticateJWTMiddleware'
+import {
+    ForbiddenError,
+    PreconditionFailedError,
+    UnauthorizedError,
+} from '../errors/HttpError'
+
+describe('authentificateJWTMiddleware', () => {
+    const mockNext = jest.fn()
+    const mockRes = jest.fn()
+
+    afterEach(() => {
+        mockNext.mockClear()
+        mockRes.mockClear()
+    })
+
+    it('should not be authorized due to missing authorization header', () => {
+        expect(() => {
+            authenticateJWTMiddleware(
+                {
+                    headers: {},
+                } as any,
+                mockRes as any,
+                mockNext
+            )
+        }).toThrowError(UnauthorizedError)
```

Comment:
I'm not sure this works. Aren't you passing a function to `expect` without executing it ? Please consider using a try/catch block here, like in other cases.

Reply:
`authenticateJWTMiddleware` function is called inside the arrow function passed to the `expect`. It makes more sense for me because you can read the test more easily as a phrase. I would have use it on other test if it was possible too.